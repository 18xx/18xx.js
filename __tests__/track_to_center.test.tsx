import Point from '../src/point';
import TrackToCenter from '../src/track_to_center';

describe('TrackToCenter', () => {
  describe('#toString()', () => {
    it('returns the svg path for the track to position 3', () => {
      expect(new TrackToCenter(3).toString()).toMatchSnapshot();
    });

    it('returns the svg path for the track to position 1', () => {
      expect(new TrackToCenter(1).toString()).toMatchSnapshot();
    });
  });
});
