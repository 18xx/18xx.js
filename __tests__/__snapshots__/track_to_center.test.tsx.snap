exports[`TrackToCenter #toString() returns the svg path for the track to position 1 1`] = `"[object Object]"`;

exports[`TrackToCenter #toString() returns the svg path for the track to position 3 1`] = `"[object Object]"`;
